packages/user/src/model/users/service.ts

```diff
@@ -78,6 +97,39 @@ class UserService {
       };
     }
   };
+
+  currentUser = async (userId: string) => {
```

Comment:
since service will not know the current user, The name of the method should be getUserById.